### PR TITLE
FH-3621 - Change the cmd  admin-auditlog to follow the V3 standard an…

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3621

**Following the changes**
- Fix filter parameters. It is woking for the type and limit now.
- Change to follow V3 standard
- Change the command name to follow project organization/structure
From: fhc admin-auditlog listlogs
To: fhc admin appstore auditlog list
- Add unit test
- Improve output
 
**Relations**
- JIRA open to change automated tests: https://issues.jboss.org/browse/FH-3752
- JIRA open to fix backend REST service to apply filters informed into the request:  https://issues.jboss.org/browse/RHMAP-16663

**Steps to verify**
- `./bin/fhc.js admin appstore auditlog list`
- `./bin/fhc.js admin appstore auditlog list --json`
- `./bin/fhc.js admin appstore auditlog list --limit=2`
- `./bin/fhc.js admin appstore auditlog list --type=android`

**Local Tests**

screen shot 2017-07-09 at 3 07 29 pm
screen shot 2017-07-09 at 3 07 02 pm
screen shot 2017-07-09 at 3 06 46 pm

### CODE REVIEW

**Comment by camilamacedo86** [comment]
@trepel jfy. In this PR will be required change the fh-art. The JIRA open for it is: https://issues.jboss.org/browse/FH-3752

File: lib/cmd/fh3/admin/appstore/auditlog/list.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        return cb(err);
+      }
+      if (!params.json) {
+        if ( data && data.list.length > 0) {
+          var headers = ["Store Item ID", "Type", "User ID", "Device ID", "Store Item Title"];
+          var fields = ["storeItemBinaryGuid", "storeItemBinaryType", "userId", "deviceId", "storeItemTitle"];
+          params._table = common.createTableFromArray(headers, fields, data.list);
+          return cb(null, params);
+        } else {
+          return cb(null, i18n._('Not found App Store audit logs'));
```
Comment (pb82) [inline lib/cmd/fh3/admin/appstore/auditlog/list.js]: Maybe change it to`No audit logs found`?

Comment (camilamacedo86) [inline lib/cmd/fh3/admin/appstore/auditlog/list.js]: 👍 Done.

**Comment by camilamacedo86** [comment]
Really Tks for the review 👍

**Pull request merged**